cli/cmd/port-forward/port_forward.go

```diff
@@ -15,7 +19,7 @@ import (
 // portForwardCmd represents the port-forward command
 var portForwardCmd = &cobra.Command{
 	Use:   "port-forward",
-	Short: "Port-forwards TimescaleDB, Promscale, Promlens, Grafana, and Prometheus to localhost",
+	Short: "Port-forwards TimescaleDB, Promscale, Promlens, Grafana, Prometheus and Jaeger to localhost",
```

Comment:
```suggestion
	Short: "Port-forwards TimescaleDB, Promscale, Promlens, Grafana, Prometheus, and Jaeger to localhost",
```

nit: oxford comma :)